templates/rolebinding.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.rbacEnabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "fullname" . }}
```

Comment:
Let's add namespace name here explicitly since it's a RoleBinding, it would associate to default namespace if not specified.

Reply:
good idea, will PR separately